src/server/handlers/dynamic.js

```diff
@@ -87,6 +81,7 @@ export default ({ server, config }) => {
         } catch (e) {
           // There has eiter been a 'natural 404', or we've thrown one
           // due to an empty response from a WP endpoint
+          log.debug('Multidata failed', e)
           componentData = e
           fetchRequestHasErrored = true
           log.error(`Endpoint data failed to fetch:`, componentData)
```

Comment:
This will duplicate the previous debug statement, probs best to remove

Reply:
:+1: probably best to remove this whole _file_ / _my writing code priveledges_

Reply:
oh cool, i can do that? removing whole file and write code privileges